tests/js/system/data/test-data-loader.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * This file is loaded by Composer autoload-dev, and that happens before `add_action` is available.
```

Comment:
I have tried to manually trigger the Monkey Brains setup method, to avoid having to use the ugly global WordPress mocking.

The following code snippet works as expected and removes all problems we have with the current approach.

```
// Load Monkey Brains WP mock methods.
Brain\Monkey\setUp();

add_action( 'admin_init', 'yoast_acf_analysis_test_data_loader', 11 );
```

Reply:
Oh, this code isn't _mocking_ `add_action` but actually adds a filter by messing with the globals before the plugin API is ready.

But I just realised that that isn't needed anyway because the autoloader only gets loaded [after the Plugin API is ready anyway](https://github.com/Yoast/yoast-acf-analysis/blob/0198d956b3ee263db0a28f0c51be41697b97e16b/yoast-acf-analysis.php#L37). So I can further simplify this.

Reply:
...and that was there for a reason because of course when running Unit Tests the Plugin API isn't ready. Your alternative approach is nice though. I just wrapped it inside a conditional now because otherwise `Brain\Monkey\setUp()` is also called during normal execution which needlessly loads Patchwork.